lib/tasks_private/release.rake

```diff
@@ -0,0 +1,37 @@
+namespace :release do
+  desc "Tasks to run on a new branch when a new branch is created"
+  task :new_branch do
+    require 'pathname'
+
+    branch = ENV["RELEASE_BRANCH"]
+    if branch.nil? || branch.empty?
+      STDERR.puts "ERROR: You must set the env var RELEASE_BRANCH to the proper value."
+      exit 1
+    end
+
+    current_branch = `git rev-parse --abbrev-ref HEAD`.chomp
+    if current_branch == "master"
+      STDERR.puts "ERROR: You cannot do new branch tasks from the master branch."
+      exit 1
+    end
+
+    root = Pathname.new(__dir__).join("../..")
+
+    # Modify settings.yml
+    settings = root.join("config", "settings.yml")
+    content = settings.read
+    settings.write(content.gsub(%r{(:docker_image: manageiq/amazon-smartstate:).+$}, "\\1latest-#{branch}"))
```

Comment:
So, we currently have the following image tags:
```
v1.0.0 - 4 years ago
latest - a year ago
latest-jansa - a year ago
latest-kasparov - 10 months ago
```
Should this be Semver or `latest-#{manageiq_version}`?

Reply:
This needs to be rebuilt for each release for now since we only have branch based releases.  Once we can detangle amazon_ssa_support into a normal gem, then I think we should switch to a semver based release.